### ISSUE TITLE
Gyro axes don't change significantly

### ISSUE THREAD

**Issue by TieSoul** [mfosse/JoyCon-Driver#68]
All gyro axes hang around 50% on my vJoy, only dipping above or below by very slight amounts even as I vigorously swing the joy-con around. When I try to use the gyro controls in dolphin, it doesn't even pick up on them.

**Comment by TieSoul** [comment]
Worth noting that the preview window and mouse controls work perfectly.

**Comment by ZenDMO** [comment]
Same here, and it had worked with previous versions from a month or so ago maybe. (Honestly probably longer before that.) It renders Gyro controls on anything other than the mouse useless as vjoy cant change the sliders more than about 1 or 2% off 50%.

**Comment by mfosse** [comment]
this is what it currently looks like in the code:

```
float mult = settings.gyroSensitivityX / 100.0f;
iReport.wAxisZRot = 16384 + (jc->gyro.roll * mult);
iReport.wSlider = 16384 + (jc->gyro.pitch * mult);
iReport.wDial = 16384 + (jc->gyro.yaw * mult);
```

I guess I forgot to update the sensitivity when I changed it everywhere else, I'll also make the sensitivity be able to be negative in the next update, which will fix issue #67
